apis/core.oam.dev/v1alpha2/core_types.go

```diff
@@ -530,7 +530,7 @@ type DependencyToObject struct {
 // +kubebuilder:object:root=true
 
 // An ApplicationConfiguration represents an OAM application.
-// +kubebuilder:resource:shortName=appconfig,categories={oam}
+// +kubebuilder:resource:shortName=appconfig;config,categories={oam}
```

Comment:
let's keep it as it is , because this CRD was not used in OAM spec 0.3. And the `config` could possiblly conflict  with other resources.

Reply:
Done